INSTALL

```diff
@@ -34,6 +34,10 @@ sudo apt install libpcre3-dev libssl-dev mysql-server mysql-client libmysqlclien
 
  sudo apt install python2.7-dev liblua5.2-dev libperl-dev
 
+* Install dependencies on Raspberry Pi OS:
+
+sudo apt install libpcre3-dev libssl-dev mariadb-server mariadb-client libmariadb-dev libmariadb-dev-compat libmaxminddb-dev libicu-dev gettext libasprintf-dev liblua5.2-dev g++ make cmake
```

Comment:
👍🏼 except installing mariadb stuff imho

Reply:
There is a short note on this page > https://github.com/Verlihub/verlihub/wiki/Installation#debian-based-distributions

````If you would like to use MariaDB instead:
sudo apt install mariadb-server libmariadb-dev mariadb-client````